Sources/Web3Auth/Provider.swift

```diff
@@ -22,6 +22,7 @@ public enum Web3AuthProvider: String, Codable {
     case WEIBO = "weibo"
     case WECHAT = "wechat"
     case EMAIL_PASSWORDLESS = "email_passwordless"
+    case WEBAUTHN = "webauthn"
```

Comment:
webauthn is not a valid provider for mobile sdks and shouldn't be invoked.

Reply:
done